Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -1375,9 +1375,15 @@ unsigned MediaPlayerPrivateGStreamerBase::decodedFrameCount() const
 {
     guint64 decodedFrames = 0;
     GstElement* sink = m_fpsSink.get();
+    GRefPtr<GstElement> scopedSink;
```

Comment:
Use just `GRefPtr<GstElement> sink` for everything, using `sink.get()` as needed in the rest of the code below to make it build in places where a `GstElement*` is needed.